### ISSUE TITLE
Add support for :cq

### ISSUE THREAD

**Issue by m1foley** [akahuku/wasavi#218]
In Vim, `:cq` works like `:qall!` except that Vim returns a non-zero exit code. A common use is when writing a git commit message, `:cq` is used to abort the commit (whereas `:q!` exits with a success code and commits with the default message).

In Wasavi, I'd expect `:cq` to behave identically to `:q!`.

**Comment by dkarter** [comment]
Agree, I found myself accidentally typing that in Wasavi. At the very least it would be nice to have `cmap` / `cnoremap` to allow adding this as an alias in the vimrc.
